src/utilities/graphql/fragments.ts

```diff
@@ -75,15 +77,16 @@ export function getFragmentQueryDocument(
     ...document,
     definitions: [
       {
-        kind: 'OperationDefinition',
-        operation: 'query',
+        kind: 'OperationDefinition' as Kind.OPERATION_DEFINITION,
+        // OperationTypeNode is an enum
+        operation: 'query' as OperationTypeNode,
```

Comment:
We could avoid having to cast by using the correct enum value:

```suggestion
        operation: OperationTypeNode.QUERY,
```

Reply:
The `OperationTypeNode` literal does not exist in GraphQL 15.